cluster-autoscaler/main.go

```diff
@@ -130,6 +130,7 @@ var (
 	maxGracefulTerminationFlag = flag.Int("max-graceful-termination-sec", 10*60, "Maximum number of seconds CA waits for pod termination when trying to scale down a node.")
 	maxTotalUnreadyPercentage  = flag.Float64("max-total-unready-percentage", 45, "Maximum percentage of unready nodes in the cluster.  After this is exceeded, CA halts operations")
 	okTotalUnreadyCount        = flag.Int("ok-total-unready-count", 3, "Number of allowed unready nodes, irrespective of max-total-unready-percentage")
+	noScaleUpFromZero          = flag.Bool("no-scale-up-from-zero", false, "Shouldn't CA scale up when there 0 ready nodes.")
```

Comment:
nit: can we change this to a positive ("scale-up-from-zero")?

Reply:
I used negative because I want unit tests to test it as default without changing tests' code.
If you think this is big problem then I can change it.

Reply:
Using positive with default value 'true' would have the same effect, right?

Reply:
Changed. I will change tests to use default values in next PR